docs/source/advanced/using-graphbolt.rst

```diff
@@ -166,3 +166,94 @@ data using our GraphBolt conversion entry point:
     # We'll see the GraphBolt representation has been re-created
     ls /tmp/acm_graphbolt/part0
     edge_feat.dgl  fused_csc_sampling_graph.pt  graph.dgl  node_feat.dgl
+
+Using GraphBolt on SageMaker
+----------------------------
+
+To prepare data to use with GraphBolt on SageMaker
```

Comment:
That should be necessary whether you are using the distributed pipeline or gconstruct?

Reply:
We can call it out that if user wants to use graphbolt, they can do all of them in one job for GConstruct, but one additional step for distributed partition pipeline.